doc/sphinx/proof-engine/tactics.rst

```diff
@@ -202,28 +196,25 @@ For a goal :g:`A \/ B`, use :tacn:`left` to replace the current goal with :g:`A`
   of :n:`let-ins`, the patterns are applied to the arguments and :n:`let-in` variables.
 
 * :n:`( {+& @simple_intropattern} )` - matches a term that consists of one or more
-  right-hand nested binary inductive types in a form such as :g:`(a1 OP (a2 OP ...))`.
+  nested binary inductive types in a right-associative form such as :g:`(a1 OP (a2 OP ...))`.
   The splitting pattern can have more than 2 names, for example :g:`(H1 & H2 & H3)`
   matches :g:`A /\ B /\ C`.
   The inductive types must have a
   :ref:`single constructor with two parameters <intropattern_cons_note>`.
   :ref:`Example <intropattern_ampersand_ex>`
 
-* :n:`[ {+| @intropatterns} ]` - splits an inductive type that has
+* :n:`[ {*| @intropatterns} ]` - splits an inductive type that has
```

Comment:
Here we go. Now that you have changed `+` in `*`, the bullet-point above is just a specialization of this case.

Reply:
Not quite.  Both the 2 preceding points should have `+`.